content/en/docs/tutorials/stateful-application/cassandra.md

```diff
@@ -53,17 +54,18 @@ To complete this tutorial, you should already have a basic familiarity with [Pod
 ### Additional Minikube Setup Instructions
 
 {{< caution >}}
-**Caution:** [Minikube](/docs/getting-started-guides/minikube/) defaults to 1024MB of memory and 1 CPU which results in an insufficient resource errors during this tutorial. 
-{{< /caution >}}
+**Caution:** [Minikube](/docs/getting-started-guides/minikube/) defaults to 1024MB of memory and 1 CPU. Running Minikube with the default resource configuration may result in insufficient resource errors during this tutorial. To avoid these errors, we recommend running Minikube with 5 GB of memory and 4 CPUs:
 
-To avoid these errors, run minikube with:
-
-    minikube start --memory 5120 --cpus=4
+```bash
+minikube start --memory 5120 --cpus=4
+```
+{{< /caution >}}
```

Comment:
This is incorrect.

Reply:
What is incorrect here?

Reply:
I don't remember. I think maybe I thought that the code block needed to be indented, but I don't think it does.

Reply:
Okay, no worries 👍